source/main/index.js

```diff
@@ -72,6 +73,8 @@ const onAppReady = async () => {
   // first checks for japanese locale, otherwise returns english
   const systemLocale = detectSystemLocale();
 
+  const stateDirectory = stateDirectoryPath;
```

Comment:
@DeeJayElly this line won't be needed once the response is changed to use `stateDirectoryPath `